css/styles.css

```
@@ -118,3 +123,32 @@ nav a {
   background: black;
   font-family: Arial, Helvetica, sans-serif;
 }
+.button_black {
```

Please try to use BEM method for naming your classes - this will give more structure and readability to your CSS! 
```suggestion
.button--black {
```